src/js/themes/base.js

```diff
@@ -510,6 +510,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         size: `${baseSpacing * 2}px`,
         // extend: undefined,
       },
+      box: {
```

Comment:
the box should be commented out since it is an empty object.

Reply:
I was following the other patterns in the theme file 

https://github.com/grommet/grommet/blob/master/src/js/themes/base.js#L649

https://github.com/grommet/grommet/blob/master/src/js/themes/base.js#L387

From what I saw the objects were not commented out even if they were empty

Reply:
hmm ok when I comment it out then it can not find the property of extend for that object `box`. 
I may be missing something but the same case is for if you comment out icon and icons within checkbox as well. 

https://github.com/grommet/grommet/blob/master/src/js/themes/base.js#L491

Reply:
Not sure if it is possible since this code has never existed before, so it should work.
The link you shared is pointing to `icon`, was that the intention? what does it mean?

I think you just need to add an existence check on your code before you are approaching `box`.
If you meant to point to:
```
${props => props.theme.checkBox.box.extend};
```
It will be something like 
```
${props => props.theme.checkBox.box & props.theme.checkBox.box.extend};
```
And it should work as expected.